indico/modules/events/editing/blueprint.py

```diff
@@ -119,15 +119,13 @@
 _bp.add_url_rule(contrib_api_prefix + '/add-paper-file', 'api_add_contribution_file',
                  timeline.RHEditingUploadContributionFile, methods=('POST',))
 _bp.add_url_rule(contrib_api_prefix + '/<int:revision_id>/review', 'api_review_editable',
-                 timeline.RHReviewEditable, methods=('POST',))
+                 timeline.RHReviewEditable, methods=('POST', 'PATCH', 'DELETE',))
```

Comment:
```suggestion
                 timeline.RHReviewEditable, methods=('POST', 'PATCH', 'DELETE'))
```

This is only needed for single-element tuples... (too bad there's no flake8 rule for this)